api/lib/middleware/proxy.ts

```diff
@@ -9,6 +9,13 @@ const logger = log4jui.getLogger('proxy')
 
 export const onProxyError = (err, req, res) => {
     logger.error(err)
+
+    if (req.baseUrl && req.baseUrl === '/activity' && req.user && req.user.userinfo) {
```

Comment:
You don't really need the initial `req.baseUrl` test because it's entirely covered by `req.baseUrl === '/activity'`.